pkg/provisioner/ironic/ironic.go

```diff
@@ -220,8 +220,8 @@ func newProvisionerWithSettings(host metal3v1alpha1.BareMetalHost, bmcCreds bmc.
 
 func newProvisionerWithIronicClients(hostData provisioner.HostData, publisher provisioner.EventPublisher, clientIronic *gophercloud.ServiceClient, clientInspector *gophercloud.ServiceClient) (*ironicProvisioner, error) {
 	// Ensure we have a microversion high enough to get the features
-	// we need.
-	clientIronic.Microversion = "1.56"
+	// we need. 1.69 adds DeploySteps to provisioning.
+	clientIronic.Microversion = "1.69"
```

Comment:
Question: could be this one considered a potential breaking change worth to be documented?

Reply:
Do we have any sort of release notes for that?

Reply:
AFAIK I don't think we have rl, and probably this could be a good point. Currently, there's just the docs in the `\docs` folder, a possibility could be to add a `Requirements` section there

Reply:
I'll put something in configuration.md